Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -302,13 +302,23 @@
         {
           "Id": "SEC101/037",
           "Name": "DoNotExposePlaintextSecrets/SqlConnectionString",
```

Comment:
>SqlConnectionString [](http://example.com/codeflow?start=47&length=19)

The ADO and JDBC rule could probably be merged into a single set of patterns. I'm avoiding that until we look at the enquoting issue, though (see comments in the shared strings file).